tools/manifest/item.py

```diff
@@ -120,6 +120,9 @@ def __init__(self,
         assert url is None or url[0] != "/"
         self._url = url
         self._extras = extras
+        parsed_url = urlparse(self.url)
+        self._flags = (set(parsed_url.path.rsplit("/", 1)[1].split(".")[1:-1])
+                       | set(parse_qs(parsed_url.query).get("wpt_flags", [])))
```

Comment:
This is a lint error, from the unittests bot:

`./manifest/item.py:125:24: W503 line break before binary operator`